### DISCUSSION TITLE
Modify multiple polygons label

### DISCUSSION THREAD

**Discussion by francesco-p** [wkentaro/labelme#1136] [Ideas / Feature requests]
Hello, 
I'm annotating a dataset for instance segmentation, I would like to change different polygons label with one action. Right now I have to manually modify the label of each polygon in an image. 

![image](https://user-images.githubusercontent.com/5663725/75090782-af0b4d00-5566-11ea-9d77-286a18f3b28b.png)

Is it any workaround for this feature? Because there is no bulk edit option...

Thanks.

**Comment by wkentaro** [comment]
Currently, there is no such feature, and I'm not sure we should add it, because we would want to modify all polygon labels from one to another for all annotated file next.
Below is my suggessted solution
```
$ cd examples/instance_segmentation
$ for f in *.json; do sed -i 's/"label": "person"/"label": "human"/g' $f; done
$ git diff
diff --git a/examples/instance_segmentation/data_annotated/2011_000003.json b/examples/instance_segmentation/data_annotated/2011_000003.json
index ef32cc3..ebb0e36 100644
--- a/examples/instance_segmentation/data_annotated/2011_000003.json
+++ b/examples/instance_segmentation/data_annotated/2011_000003.json
@@ -3,7 +3,7 @@
   "flags": {},
   "shapes": [
     {
-      "label": "person",
+      "label": "human",
...
```

**Comment by francesco-p** [comment]
Humm...I see your workaround, but this assumes that you convert all the polygons with labels X in label Y. Instead what I want, is to modify an arbitrary number of polygons with label X to polygon with label Y and maybe Z.
Thanks anyway, I think it would be a cool feature.
